src/pages/MyPage.js

```diff
@@ -91,7 +84,7 @@ const MyPage = () => {
           <NicknameTitle>닉네임</NicknameTitle>
           <EditNickname userNickname={userInfo.userNickname} />
           <SchoolAuthTitle>학교인증</SchoolAuthTitle>
-          <SchoolAuth />
+          <SchoolAuth is_auth={userInfo.is_auth} />

```

Comment:
위의 이유로 여기도 바뀌면 좋겠네요~

```suggestion
          <SchoolAuth isAuth={userInfo.is_auth} />

```